manifests/t/Texlive/Texlive/2023/Texlive.Texlive.installer.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Texlive.Texlive
+PackageVersion: 2023
+Installers:
+- InstallerUrl: https://mirrors.rit.edu/CTAN/systems/texlive/tlnet/install-tl-windows.exe

```

Comment:
```suggestion
- InstallerUrl: https://mirror.ctan.org/systems/texlive/tlnet/install-tl-windows.exe
```

Reply:
Sorry to keep changing this, but based on our conversation on the other change, this might be a better URL. Hopefully it will retain the same geo-routing within WinGet as in browsers. What do you think?

Reply:
I think you're right, ctan.org is after a well-known and trusted domain among LaTeX users